hub/policy/src/main/java/uk/gov/ida/hub/policy/domain/IdpSelected.java

```diff
@@ -23,10 +25,19 @@ public IdpSelected(String selectedIdpEntityId, String principalIpAddress) {
         this.principalIpAddress = principalIpAddress;
     }
 
+    // TODO: Remove this construtor after the corresponding hub release
     public IdpSelected(String selectedIdpEntityId, String principalIpAddress, Boolean registration) {
```

Comment:
not worth changing (as the code is about to be deleted), but for the future, a slightly better pattern for this is

```
public Foo(Thing a, Thing b) {
    return new Foo(a, b, null);
}

public Foo(Thing a, Thing b, Thing c) {
    this.a = a;
    this.b = b;
    this.c = c;
}
```

That way we only have to do the setting once, we cut down the code duplication, and the intention is a little clearer.